client/src/components/util/graphql/Mutations.ts

```diff
@@ -72,3 +95,9 @@ export const UPDATE_REQUEST = gql`
         }
     }
 `;
+
+export const DELETE_REQUEST = gql`
```

Comment:
Non-admins can't delete requests, btw, and it's highly discouraged anyways.  We have CANCELLED and ABANDONED statuses which are like soft-deletes